__tests__/app.test.js

```diff
@@ -40,4 +40,32 @@ describe('endpoint GET /api',()=>{
             expect(body.endpoints).toEqual(endpointsObject)
         })
     })
+})
+
+describe('endpoint GET /api/articles/:article_id',()=>{
+    test('receives 200 response and returns correct article',()=>{
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.article.article_id).toBe(2)
+            expect(Object.keys(body.article).length).toBe(8)
+        })
```

Comment:
Please check the entire contents of the object. We know what will come back as we have asked for a specific article :)